data/card_catalogue/weapons.json

```diff
@@ -5,38 +5,37 @@
     "tier": "green",
     "details": {
       "rules": {
-        "passive": "Melee attacks knock the target back 3 squares",
-        "active": "Throw a heavy sledgehammer at a visible target"
+        "passive": "Melee attacks knock the target back 3 metres"
       },
       "flavour": "This one'll bring down the house.",
-      "image": "assets/core/card/illustration/Bean Blast.png"
+      "image": "assets/core/card/illustration/Sledgehammer.png"
     },
     "draft": true
   },
   {
-    "title": "Bow Charm",
+    "title": "Bow Hex",
     "upgrade": "",
     "tier": "green",
     "details": {
       "rules": {
-        "passive": "Roll an additional d4 when performing a ranged attack",
-        "active": "Deal 10 damage to a visible target."
+        "active": "Deal 10 damage to a visible target.",
+        "passive": "Roll an additional d4 when performing a ranged attack."
       },
       "flavour": "They stood on the hill for a moment, mesmerized by the dazzling light",
-      "image": "assets/core/card/illustration/Bow Charm.png"
+      "image": "assets/core/card/illustration/Bow Hex.png"
     }
   },
   {
-    "title": "Longsword",
+    "title": "Sword Hex",
```

Comment:
Re-titled this one and Bow Charm to match each other. Will be linking this commit to #105 as a reminder to make their art similar to one another, as per 340534f. I'll be addressing the weapon situation more generally in https://github.com/elliottomlinson/rpcg/issues/112

Reply:
Left some comments in there, I'm good to merge this in and revisit weapons as a whole separately